Automattic/jetpack
# Jetpack Social: Add notice to promote Instagram

## Proposed changes:

This PR adds a notice to the Jetpack Social admin page, as well as the Jetpack Social sidebar, to promote that Instagram is now available.

- On the admin page, the text as well as primary button change depending on whether or not the user has a paid plan.
- Dismissing one notice should prevent the other one from showing.

**Note:** We cannot merge this until we have implemented a proper feature check for Instagram.

### Other information:


## Jetpack product discussion


pdrWKz-XH-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:





* Create a new site with Jetpack Social.
* Open the Social admin page and verify that you don't see a notice.
* Add support for Instagram connections to your site: D111381-code
* Refresh and make sure there notice is there.

* Dismiss the notice, refresh the page and confirm that the notice is missing.
* Reset the notice: `jetpack docker wp option delete jetpack_social_dismissed_notices`
* Refresh and make sure it appears again.
* Buy the Advanced plan and make sure the notice has different text and CTA.



* Create a new post, go to the Jetpack Social sidebar.
* Check that the notice is there and dismissable.
* Make sure the notices match the designs.

![Screenshot 2023-05-17 at 16 49 25@2x](https://github.com/Automattic/jetpack/assets/1713699/ec586fc3-e57b-4dc7-92b2-c2ddd24227af)

<img width="662" alt="Screenshot 2023-05-17 at 16 48 36@2x" src="https://github.com/Automattic/jetpack/assets/1713699/3ef120dc-e954-462c-a9f8-5b2270d0d96b">
